src/ducks/balance/components/GroupPanel.jsx

```diff
@@ -154,7 +154,7 @@ class GroupPanel extends React.PureComponent {
             disableRipple: true
           }}
           className={cx({
-            [styles['GroupPanelSummary--unchecked']]: !checked
+            [styles['GroupPanelSummary--unchecked']]: !checked && !group.loading
```

Comment:
Maybe this class name is not relevant anymore if it's used not only to show that a panel is unchecked, but also when it's loading.

Reply:
I create a variable `const isUncheckable = !group.loading` which allow to keep a coherence